includes/wikia/nirvana/WikiaView.class.php

```diff
@@ -31,16 +31,12 @@ function __construct( WikiaResponse $response = null ) {
 	 * @param string $methodName
 	 * @param array $data
 	 * @param string $format
+	 *
+	 * @return WikiaView
 	 */
-	public static function newFromControllerAndMethodName( $controllerName, $methodName, Array $data = array(), $format = WikiaResponse::FORMAT_HTML ) {
-		$app = F::app();
+	public static function newFromControllerAndMethodName( $controllerName, $methodName, Array $data = [], $format = WikiaResponse::FORMAT_HTML ) {
 		// Service classes must be dispatched by full name otherwise we default to a controller.
```

Comment:
The following 6 lines were functionally duplicated later in code so replaced by common method ```normalizeControllerClass```